package.json

```diff
@@ -37,9 +37,9 @@
   "dependencies": {
     "cross-fetch": "^3.0.4",
     "universal-cookie": "^4.0.3",
-    "vue-apollo": "^3.0.3",
+    "vue-apollo": "3.0.4",
```

Comment:
Why is the range to the latest version removed?

Reply:
Because they have introduced a breaking change (only with typescript) in a patch ...

Btw, it's a good practice to fix version and to manually check/tests for packages updates. 

I can change it if you want.

Reply:
It's just that there will be duplicate dependencies. Since if vue-apollo is installed manually for types eg.

Reply:
Good point ! That's done

Reply:
ping @negezor